src/controllers/user.ts

```diff
@@ -0,0 +1,44 @@
+import Users from '../models/users';
+
+interface userDetails {
+  firstName: string;
+  lastName: string;
+  phone: string;
+  bvn: string;
+  email: string;
+  password: string;
+}
+
+export async function getAllUsers() {
+  return Users.find({ deletedAt: null }).sort({ firstName: 'asc' });
+}
+
+export async function getAuser(userID: string) {
```

Comment:
hello Usman, I'll edit the name of this function to `getAUser` if that's okay with you

Reply:
Yea. I noticed it, you can change it.